spanner/client.go

```diff
@@ -331,6 +335,12 @@ type ClientConfig struct {
 	DirectedReadOptions *sppb.DirectedReadOptions
 
 	OpenTelemetryMeterProvider metric.MeterProvider
+
+	// EnableServerSideTracing indicates whether server side tracing is enabled or not.
+	// If it is enabled, trace spans will be created at Spanner layer.
+	//
+	// Default: false
+	EnableServerSideTracing bool
```

Comment:
Do we need an ENV also here similar to what is being done in Java?
SPANNER_ENABLE_SERVER_SIDE_TRACING

Reply:
No, we don't ENV here. Confirmed with Simil.

Reply:
No, for JAVA we are keeping to make consistency with other options. For Go, discussed with Simil earlier and we thought that we can add this incrementally if needed.